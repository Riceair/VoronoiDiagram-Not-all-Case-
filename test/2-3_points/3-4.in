#三點測試 銳角三角形
3
147 190
164 361
283 233