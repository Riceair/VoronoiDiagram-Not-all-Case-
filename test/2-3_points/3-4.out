P 147 190
P 164 361
P 283 233
E -107534 10981.47 196.05 271.47
E 196.05 271.47 27286.05 -85408.53
E 196.05 271.47 80836.05 75241.47
